packages/paste-website/src/pages/components/chat-composer/index.mdx

```diff
@@ -0,0 +1,250 @@
+## Guidelines
+
+### About Chat Composer
+
+A Chat Composer is an input made for users to type rich chat messages. Chat Composer is best used as one part of larger chat user interface to provide a seamless authoring experience.
+Within the context of Paste, Chat Composer is most typically used alongside the [`Chat Log`](/components/chat-log) component.
+
+### Accessibility
+
+Chat Composer supports a variety of aria attributes which are passed into the content editable region of the component. If the surrounding UI includes a screen reader visible label
+reference the label element using `aria-labelledby`. If the surrounding UI does not include a screen reader visible label, use `aria-label` to describe the input. If the surrounding UI
+includes additional help or error text use `aria-describedby`.
+
+### Lexical and plugins
+
+Chat Composer is built on top of the [Lexical](https://lexical.dev) editor. Lexical is extensible and follows a declarative approach to configuration via JSX. Developers can leverage a
+wide variety of [existing plugins](https://github.com/twilio-labs/paste/blob/main/packages/paste-libraries/lexical/src/index.tsx) via the `@twilio-paste/lexical-library` package or other
+sources. Alternatively, developers can write their own custom plugin logic. Plugins are provided to the Chat Composer via the `children` prop.
+
+#### Auto Link Plugin
+
+Chat Composer uses a custom [`AutoLinkPlugin`](https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/chat-composer/src/AutoLinkPlugin.tsx) internally
+which you can see being configured [here](https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/chat-composer/src/ChatComposer.tsx#L116) as a JSX child.
+
+## Examples
+
+### With placeholder
+
+Set a placeholder value using a `placeholder` prop.
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'customer-chat', onError: (e) => { throw e } }} placeholder="Chat text" ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With initial value
+
+Set an initial value using an `initialValue` prop.
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'customer-chat', onError: (e) => { throw e } }} initialValue="This is my initial value" placeholder="Type Here..." ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With max height
+
+Restrict the height of the composer using a `maxHeight` prop.
+
+<LivePreview scope={{ChatComposer, tallInitialText}} language="jsx">
+  {`<ChatComposer config={{editorState: tallInitialText, namespace: 'customer-chat', onError: (e) => { throw e } }} maxHeight="size10" ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With rich text
+
+Set a rich text value using one of the Lexical formatting APIs such as [`toggleFormat`](https://lexical.dev/docs/api/classes/lexical.TextNode#toggleformat)
+
+<LivePreview scope={{ChatComposer, richInitialText}} language="jsx">
```

Comment:
suggestion: I like how clean the codeblocks are, but it may be unclear if folks can't see the `richInitialText` code in this sample. Any chance we can inline it?